### DISCUSSION TITLE
Overheating issue with SMPlayer when not closed

### DISCUSSION THREAD

**Discussion by KrNeilesh** [smplayer-dev/smplayer#1078] [Ideas]
Overheating issue with SMPlayer when not closed. PC do not go to sleep mode and gets overheated. Using Dell Vostro 5481 with Nvidia GE Force

**Comment by bendover22** [comment]
OS?
Overheating what - GPU, CPU...?
What's meant by "issue w/ smp when (what) not closed?

If * are playing A/V w/ smp, why should the pc go to sleep?  Or do you mean screensaver doesn't activate?
If *not playing A/V files, smp has nothing to do w/ it.

"Sleep mode" (or pwr mgr app in an most OSes) isn't based on hardware temps, or vice versa.
If have a discreet Nvidia GPU. it's cooling may not be working 100% (dirty fan, cooling fins, etc.).  I have to clean GPU fan, fins every few months.

If CPU's overheating, are you using **default** SMP video / quality settings, or custom video settings?  Like passing **--scale=** options to mpv / mplayer, that (can) take a lot more resources (maybe both CPU & GPU).
